tests/common_test.py

```diff
@@ -60,7 +60,7 @@ def fixed_point_fun(params, *args, **kwargs):
 
       hash(solver)  # Checks that hash works.
       update = jax.jit(solver.update)
-      state0 = solver.init_state(params0)
+      state0 = solver.init_state(params0, data=data)
```

Comment:
this had not been spotted before because the init state methods were not computing the value function most of the solvers not requiring it in the absence of auxiliary value